include/onnxruntime/core/providers/coreml/coreml_provider_factory.h

```diff
@@ -49,6 +49,9 @@ static const char* const kCoremlProviderOption_MLComputeUnits = "MLComputeUnits"
 static const char* const kCoremlProviderOption_ModelFormat = "ModelFormat";
 static const char* const kCoremlProviderOption_RequireStaticInputShapes = "RequireStaticInputShapes";
 static const char* const kCoremlProviderOption_EnableOnSubgraphs = "EnableOnSubgraphs";
+static const char* const kCoremlProviderOption_SpecializationStrategy = "SpecializationStrategy";
+static const char* const kCoremlProviderOption_ProfileComputePlan = "ProfileComputePlan";
+static const char* const kCoremlProviderOption_AllowLowPrecisionAccumulationOnGPU = "AllowLowPrecisionAccumulationOnGPU";
```

Comment:
Is there documentation on how/when/why these options should be used and what is valid input for them?

Reply:
Add some notes from https://developer.apple.com/documentation.
But it's not that precise so far to indicate how much gains it will take.